test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -317,18 +309,15 @@ var _ = SIGDescribe("gpu Upgrade [Feature:GPUUpgrade]", func() {
 				framework.ExpectNoError(masterUpgrade(f, target, nil))
 				framework.ExpectNoError(checkMasterVersion(f.ClientSet, target))
 			}
-			runUpgradeSuite(f, gpuUpgradeTests, testFrameworks, testSuite, upgrades.ClusterUpgrade, upgradeFunc)
+			runUpgradeSuite(f, gpuUpgradeTests, testSuite, upgrades.ClusterUpgrade, upgradeFunc)
 		})
 	})
 })
 
-// TODO(#98326): Figure out the right SIG and use appropriate SIGDescribe.
+// TODO(#98326): Move the test to sig-aps dir and use SIGDescribe.
 var _ = ginkgo.Describe("[sig-apps] stateful Upgrade [Feature:StatefulUpgrade]", func() {
 	f := framework.NewDefaultFramework("stateful-upgrade")
 
-	// Create the frameworks here because we can only create them
-	// in a "Describe".
-	testFrameworks := createUpgradeFrameworks(statefulsetUpgradeTests)
```

Comment:
@wojtek-t maybe this comment means these framework creations should be put back?

Reply:
Those were already put back in subsequent PRs.

You can take a look at this comment:
https://github.com/kubernetes/kubernetes/issues/103697#issuecomment-880148377